localstack-core/localstack/utils/event_matcher.py

```diff
@@ -0,0 +1,44 @@
+# localstack/utils/event_matcher.py
+import json
+from typing import Any, Dict, Union
+from localstack import config
+from localstack.services.events.event_ruler import matches_rule
+from localstack.services.events.v1.utils import matches_event as python_matches_event
```

Comment:
Minor: We could consider grouping the Java and Python implementations because they are now spread across three different directories.